src/main/kotlin/app/revanced/patches/reddit/ad/general/fingerprints/AdPostFingerprint.kt

```diff
@@ -0,0 +1,12 @@
+package app.revanced.patches.reddit.ad.general.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+
+object AdPostFingerprint : MethodFingerprint(
+    "V",
+    // "children" are present throughout multiple versions
+    // "uxExperiences" might not work on all versions
+    // Not sure how to refine the search better.
```

Comment:
Can `uxExperiences` be removed while still allowing the fingerprint to be unique?